ts/features/itwallet/issuance/screens/ItwIssuanceEidFailureScreen.tsx

```diff
@@ -44,6 +44,13 @@ export const ItwIssuanceEidFailureScreen = () => {
     }
   };
 
+  const retryIssuance = (errorConfig?: KoState) => {
+    machineRef.send({ type: "retry" });
+    if (errorConfig) {
```

Comment:
Do we really need this `errorConfig` and conditional logic if we never use it on line 113?

Reply:
After reviewing how the KO tracking had been handled, I thought I could refactor it by improving the tracking handling and adding some events that had not been tracked even though they were present in the document.
Fixed [here](https://github.com/pagopa/io-app/pull/6332/commits/d0948963c751dd54176cb6d8a34a244c7bec3efe)

Reply:
After reviewing how the KO tracking had been handled, I thought I would refactor it by improving the tracking handling and adding some events that were not tracked even though they were requested in the document.